### DISCUSSION TITLE
backup gsettings  values

### DISCUSSION THREAD

**Discussion by libialany** [lennart-rth/Live-Earth-Wallpapers#17] [Ideas]
Configuring GNOME at low level,reading a bit of this showed me the importance of the changes, we should keep the previous configurations we had.
`gsettings list-recursively >      {path-for my-text-file-of-my-previus-configurations}`

**Comment by libialany** [comment]
can i contribute?

**Reply by lennart-rth** [reply]
You are welcome to make a PR
